podinfo/podinfo-hpa-custom.yaml

```diff
@@ -1,11 +1,11 @@
 ---
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2beta1
```

Comment:
Why the downgrade?

Reply:
see https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics

Reply:
The docs are with v2beta2 and you've changed to v2beta1, why?

Reply:
Very few changes Haha